make/Makefile.container.mk

```diff
@@ -8,7 +8,10 @@
 	@echo Preparing container image files
 	@mkdir -p ${OUTDIR}/docker
 	@cp -r deploy/docker/* ${OUTDIR}/docker
-	@cp ${GOPATH}/bin/kiali ${OUTDIR}/docker
+	for arch in amd64 arm64; do \
+  	  GOOS=linux GOARCH=$${arch} make build; \
+  	  mv ${GOPATH}/bin/kiali ${OUTDIR}/docker/kiali-$${arch}; \
+  	done
```

Comment:
is this going to run "make build" multiple times? Is there any way we can default to NOT do a multi-arch build by default? Devs doing this aren't going to want to run multi-arch builds in their code-test cycles. Multi-arch builds should only be "turned on" during CI or release builds.

Also, never use `make` directly in a Makefile. Use `$(MAKE)`.

Finally, as a general "something doesn't look right" - why is "make build" being run here inside `.prepare-kiali-image-files` anyway? The targets in `Makefile.container.mk` should assume the builds are done - builds are done in `make/Makefile.build.mk` I believe. I haven't looked closely at this yet, but I suspect there is a better way to implement this to be more consistent with the way the makefiles are designed to work.

Reply:
For multi arch images build. We need multi binaries for each arch `kiali-arm64` and `kiali-arm64`. 
so have build for multi-times with the arch suffix.

we could set `TARGET_ARCH ?= amd64` and `TARGET_ARCH="amd64 arm64"` in ci

We could add `make build-linux` in `make/Makefile.build.mk`
```
build-linux:
	for arch in ${TARGET_ARCH}; do \
      	    GOOS=linux GOARCH=$${arch} $(MAKE) build; \
            mv ${GOPATH}/bin/kiali ${GOPATH}/bin/kiali-$${arch}; \
        done
```

then final script in ci could be.
```
TARGET_ARCH="amd64 arm64" DORP=dockerx make build-linux container-build-kiali container-push-kiali-quay 
```

Reply:
@jpkrohling Updated changes i describe.

Reply:
I think you meant @jmazzitelli :-)

Reply:
@jpkrohling sorry. 

btw. i think Makefiles in kiali project are good examples to switch different docker image build tools. 

if you like this way too. i could update it for jaeger